src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/SpanNDExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Numerics.Tensors
+{
+    public static class SpanNDExtensions
```

Comment:
We put them in System just to match the existing Index/Range. I think if we want them in the same namespace we should move all to System.Numerics.Tensors. I don't think there will be any non System.Numerics consumers? @tannergooding can you think of any?

Reply:
This was part of the pending discussion from API review. There wasn't concensus yet on where `NativeIndex`, `NativeRange`, and any new span types would live.

It could be an existing general-purpose namespace like `System` or `System.Buffers`, it could be some new namespace like `System.Memory`, and it could simply be something domain specific like `System.Numerics.Tensors`. It likely also depends, in part, about the name used and how general purpose we want these to be (and whether there will correspondingly be something like `NativeSpan` which is completely independent).

Reply:
So I'll leave them where they are for now then since we need to wait for the result of the API review discussion to finalize their location.